internal/csi/driver/camanager.go

```diff
@@ -0,0 +1,132 @@
+/*
+Copyright 2021 The cert-manager Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package driver
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"time"
+)
+
+// manageCAFiles subscribes to events from the Root CAs provider, and updates
+// all managed volumes CA files accordingly. Exits early if rootCAs is not
+// configured.
+func (d *Driver) manageCAFiles(ctx context.Context, updateRetryPeriod time.Duration) {
+	log := d.log.WithName("ca-manager")
+
+	// Exit straight away if root CAs haven't been configured.
+	if d.rootCAs == nil {
+		log.Info("not running CA file manager, root CA certificates not configured")
+		return
+	}
+
+	watcher := d.rootCAs.Subscribe()
+
+	log.Info("starting root CA file manager")
```

Comment:
Note: looking at the code I see that root CA file manager would dynamically update the mounted CAs if the  root CA was changed (renewed etc). I wouldn't have thought that is the case from Readme- if so it may be worth adding that to the Readme.

Reply:
Thanks, that is what it does so will update the README.md